atomic_reactor/inner.py

```diff
@@ -643,10 +643,13 @@ def process_keyvals(keyvals):
     try:
         build_result = dbw.build_docker_image()
     except Exception as e:
+        logger.info("Dockerfile used for build: \n%s", dbw.builder.original_df)
         logger.error('image build failed: %s', e)
         raise
     else:
         if not build_result or build_result.is_failed():
+            logger.info("Dockerfile used for build: \n%s", dbw.builder.original_df)
```

Comment:
These both loggers can be extracted as one right after `else:`

Reply:
Edited ^

Reply:
right will fix